src/encoded/static/components/item-pages/MicroscopeConfigurationView.js

```diff
@@ -0,0 +1,412 @@
+        return {
+            'tab' : <span><i className="icon icon-microscope fas icon-fw"/> MicroMeta</span>,
+            'key' : 'micrometa',
+            //'disabled' : !Array.isArray(context.experiments),
+            'content' : (
+                <div className="overflow-hidden">
+                    <h3 className="tab-section-title">
+                        <span>4DN MicroMeta</span>
+                    </h3>
+                    <hr className="tab-section-title-horiz-divider"/>
+                    <MicroMetaTabView {...props} />
+                </div>
+            )
+        };
+    }
+
+    constructor(props){
+        super(props);
+        this.fullscreenButton = this.fullscreenButton.bind(this);
+        this.saveButton = this.saveButton.bind(this);
+        this.cloneButton = this.cloneButton.bind(this);
+        this.havePermissionToEdit = this.havePermissionToEdit.bind(this);
+        this.handleFullscreenToggle = this.handleFullscreenToggle.bind(this);
+        this.handleSave = _.throttle(this.handleSave.bind(this), 3000);
+        this.onSaveMicroscope = this.onSaveMicroscope.bind(this);
+        this.getMicroscopyMetadataToolComponent = this.getMicroscopyMetadataToolComponent.bind(this);
+
+        this.state = {
+            'mounted'               : false,
+            'saveLoading'           : false,
+            'cloneLoading'          : false,
+            'releaseLoading'        : false,
+            'addFileLoading'        : false,
+            'modal'                 : null
+        };
+
+        this.microMetaToolRef = React.createRef();
+    }
+
+    componentDidMount(){
+        const onComplete = () => {
+            this.setState({ mounted: true });
+        };
+
+        if (!MicroscopyMetadataTool) {
+            setTimeout(()=>{
+                // Load in HiGlass libraries as separate JS file due to large size.
+                // @see https://webpack.js.org/api/module-methods/#requireensure
+                require.ensure(['4dn-microscopy-metadata-tool'], (require) => {
+                    MicroscopyMetadataTool = require('4dn-microscopy-metadata-tool').default;
+                    onComplete();
+                }, "microscopy-metadata-bundle");
+            });
+        } else {
+            onComplete();
+        }
+    }
+
+    getMicroscopyMetadataToolComponent(){
+        return (this.microMetaToolRef && this.microMetaToolRef.current && this.microMetaToolRef.current) || null;
+    }
+
+    havePermissionToEdit(){
+        const { session, context : { actions = [] } } = this.props;
+        return !!(session && _.findWhere(actions, { 'name' : 'edit' }));
+    }
+
+    /**
+        * Update the current higlass viewconfig for the user, based on the current data.
+        * Note that this function is throttled in constructor() to prevent someone clicking it like, 100 times within 3 seconds.
+        * @returns {void}
+    */
+    handleSave(evt) {
+        const { href, context } = this.props;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['href' is assigned a value but never used. (no-unused-vars)](https://app.codacy.com/app/4dn/fourfront/pullRequest?prid=4271272)